### PR TITLE
Introducing Timestamp-task 1.0 from samples

### PR DESCRIPTION
**Pull request by cppwfs**
This will be the replacement for the timestamp-task from Spring-Cloud-Task-Starters

### CODE REVIEW

File: timestamp-task/README.adoc [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+[source,shell,indent=2]
+----
+./mvnw clean package
+----
+
+=== Build Docker Container
+
+[source,shell,indent=2]
+----
+./mvnw clean package jib:build
```
[the remark this answers is not in the document]
Comment (cppwfs) [reply timestamp-task/README.adoc:24]: Good catch.  This PR was made prior to our latest updates.   I'll refresh.

File: timestamp-task/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties [hunk 2]
```diff
@@ -0,0 +1 @@
+configuration-properties.classes=io.spring.timestamp.TimestampTaskProperties
```
[the remark this answers is not in the document]
Comment (cppwfs) [reply timestamp-task/src/main/resources/META-INF/spring-configuration-metadata-whitelist.properties:1]: Great question!   Previously we needed an additional file to expose public properties as a workaround for a missing feature in Spring Boot.    That properties file is no longer needed.

File: timestamp-task/README.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (bono007) [reply timestamp-task/README.adoc]: I went ahead and committed the changes to make it look like scenario-task (hope you don't mind )

File: timestamp-task/src/main/java/io/spring/timestamp/TimestampTaskProperties.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+package io.spring.timestamp;
+
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.util.Assert;
+
+/**
+ * @author Glenn Renfro
+ */
+@ConfigurationProperties
```
Comment (bono007) [inline timestamp-task/src/main/java/io/spring/timestamp/TimestampTaskProperties.java:25]: @cppwfs before this was prefixed at `timestamp` and now its ``. I would favor using a prefix as it's a best practice (and ppl will copy/pasta this I am sure). Wdyt?

**Pull request merged**